### PR TITLE
[485316]: HW1

### PR DESCRIPTION
**Pull request by jurf**
Sorry for the delay; I completely forgot about this assignment.

### CODE REVIEW

**Comment by jurf** [comment]
Added comment as discussed.

**Comment by ph4r05** [comment]
Hi @jurf 

> Create a new PR in the open-source-development-course-hw02-1, the previous PR has to be left intact.

Where is pls the other PR?

**Comment by ph4r05** [comment]
Btw congrats, you are the only one with Travis actually executing something.

**Comment by jurf** [comment]
Looks like I completely misunderstood the last instruction of step 1:

> Please, Do not look at the step 2 before finishing step 1. We want to see the conflicts! >:)

By ‘see’ I thought you wanted to actually look at the PR before I was allowed to continue; I uploaded this PR and just waited for comments. I’ll finish this by tomorrow, sorry.

**Comment by ph4r05** [comment]
ok cool! thanks!

**Review by ph4r05 — DISMISSED** [review]
👍

**Review by ph4r05 — CHANGES_REQUESTED** [review]
Repo is set up in a way merge is blocked for unsigned commits, GitHub warns you about this. pls fix it (sign commit)

**Comment by jurf** [comment]
I must’ve forgotten to set up git again when changing folders, will do.

**Comment by jurf** [comment]
Done.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Module",
+            "type": "python",
+            "request": "launch",
+            "module": "ossdev"
+        }
+    ]
+}
```
Comment (ph4r05) [inline .vscode/launch.json:14]: I am not familiar with tool creating this file. Is it common to version it? I mean `.idea` project files should not be in the repository so just checking if this is the case.

**Review by ph4r05 — APPROVED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (jurf) [reply .vscode/launch.json:14]: I guess it depends on the preferred workflow of the main developers, but afaik yes, it is a shared configuration instructing VS Code how to launch/debug the project, making it work basically out-of-the-box in it.

**Pull request closed without merging**